docs/source/user-guide/concepts.rst

```diff
@@ -6,6 +6,67 @@ Concepts
    :local:
    :depth: 1
 
+.. image:: /img/installing-with-conda.png
+    :align: right
+
+.. _installing-with-conda:
+
+Installing with conda
+=====================
+
+Conda install
+-------------
+* When you `conda install <package>` the happy path (with no dependencies) is that conda:
```

Comment:
I think `the happy path` is a little confusing here, that's my bad. I think what it really means is "when you install a package that exists in your channels and has no dependencies"